src/index.js

```diff
@@ -24,6 +27,9 @@ function splitGuide() {
 }
 
 function deletePreviouslyGeneratedFiles() {
+  if (argv.noclean) {
```

Comment:
I'd prefer to have this flag passed as an options argument to the splitGuide function